frontend/src/components/NotesDialog.js

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Button from "@material-ui/core/Button"
+import Dialog from "@material-ui/core/Dialog"
+import DialogActions from "@material-ui/core/DialogActions"
+import DialogContent from "@material-ui/core/DialogContent"
+import DialogContentText from "@material-ui/core/DialogContentText"
+import DialogTitle from "@material-ui/core/DialogTitle"
+import TextField from "@material-ui/core/TextField"
+
+function NotesDialog({ visibleDialog, toggleVisibleDialog }) {
+  const [participantNotes, setParticipantNotes] = React.useState("")
+
+  const handleSubmit = () => {
+    // Make POST request to API here to submit data
+    // Need to pass in visit ID to this component (most likely) for API
+    toggleVisibleDialog()
```

Comment:
@jkeary We still need a way to hit the API to make this actually functional. Are we doing this anywhere else in the app currently?

Reply:
@marcelkooi someone is actually currently working on just that.  Here's the ticket: https://github.com/CodeForPhilly/prevention-point/issues/154.  I also have as part of that ticket, hooking up your modal.

Reply:
Awesome, thanks @jkeary